docs/getting-started/first-runbook-run/define-the-runbook-process.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Defining the runbook process for workers
+description: Step by step guide on how to define a runbook process to run on Workers in Octopus Deploy.
+position: 40
+hideInThisSection: true
+---
+
+The runbook process is the steps the Octopus Server orchestrates to perform various tasks on your infrastructure.  For our simple hello world script, we will only have one step.
```

Comment:
Since you have a separate step for running a runbook on a target I think a sentence here to explain about this step running on a worker might be helpful?

Reply:
Fair point, updating